deepchecks/core/serialization/check_result/html.py

```diff
@@ -0,0 +1,325 @@
+
+
+class CheckResultSerializer(HtmlSerializer['check_types.CheckResult']):
+    """Serializes any CheckResult instance into HTML format.
+
+    Parameters
+    ----------
+    value : CheckResult
+        CheckResult instance that needed to be serialized.
+    """
+
+    def __init__(self, value: 'check_types.CheckResult', **kwargs):
+        if not isinstance(value, check_types.CheckResult):
+            raise TypeError(
+                f'Expected "CheckResult" but got "{type(value).__name__}"'
+            )
+        self.value = value
+
+    def serialize(
+        self,
+        output_id: t.Optional[str] = None,
+        check_sections: t.Optional[t.Sequence[CheckResultSection]] = None,
+        full_html: bool = False,
+        include_requirejs: bool = False,
+        include_plotlyjs: bool = True,
+        connected: bool = True,
+        **kwargs
+    ) -> str:
+        """Serialize a CheckResult instance into HTML format.
+
+        Parameters
+        ----------
+        output_id : Optional[str], default None
+            unique output identifier that will be used to form anchor links
+        check_sections : Optional[Sequence[Literal['condition-table', 'additional-output']]], default None
+            sequence of check result sections to include into the output,
+            in case of 'None' all sections will be included
+        full_html : bool, default False
+            whether to return a fully independent HTML document or only CheckResult content
+        include_requirejs : bool, default False
+            whether to include requirejs library into output or not
+        include_plotlyjs : bool, default True
+            whether to include plotlyjs library into output or not
+        connected : bool, default True
+            whether to use CDN to load js libraries or to inject their code into output
+
+        Returns
+        -------
+        str
+        """
+        if full_html is True:
+            include_plotlyjs = True
+            include_requirejs = True
+            connected = False
+
+        sections_to_include = verify_include_parameter(check_sections)
+        sections = [self.prepare_header(output_id), self.prepare_summary()]
+
+        if 'condition-table' in sections_to_include:
+            sections.append(''.join(self.prepare_conditions_table(output_id=output_id)))
+
+        if 'additional-output' in sections_to_include:
+            sections.append(''.join(self.prepare_additional_output(output_id)))
+
+        plotlyjs = plotlyjs_script(connected) if include_plotlyjs is True else ''
+        requirejs = requirejs_script(connected) if include_requirejs is True else ''
+
+        if full_html is False:
+            return ''.join([requirejs, plotlyjs, *sections])
+
+        # TODO: use some style to make it pretty
+        return textwrap.dedent(f"""
+            <html>
+            <head><meta charset="utf-8"/></head>
+            <body>
+                {''.join([requirejs, plotlyjs, *sections])}
```

Comment:
we should put requirejs and plotlyjs in the head for standard?

Reply:
what it will change? 
also putting a script tag into the head is not a standard practice as from my point/